<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>***********************************************************************
*
* Copyright (c) 2012-2023 Barbara Geller
* Copyright (c) 2012-2023 Ansel Sermersheim
*
* Copyright (C) 2015 The Qt Company Ltd.
* Copyright (c) 2012-2016 Digia Plc and/or its subsidiary(-ies).
* Copyright (c) 2008-2012 Nokia Corporation and/or its subsidiary(-ies).
*
* This file is part of CopperSpice.
*
* CopperSpice is free software. You can redistribute it and/or
* modify it under the terms of the GNU Lesser General Public License
* version 2.1 as published by the Free Software Foundation.
*
* CopperSpice is distributed in the hope that it will be useful,
* but WITHOUT ANY WARRANTY; without even the implied warranty of
* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
*
* https://www.gnu.org/licenses/
*
***********************************************************************</comment>
 <class>PhraseBookBox</class>
 <widget class="QDialog" name="PhraseBookBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Phrase Book</string>
  </property>
  <property name="whatsThis">
   <string>This window allows you to add, modify, or delete entries in a phrase book.</string>
  </property>
  <layout class="QHBoxLayout" name="hboxLayout">
   <item>
    <layout class="QVBoxLayout" name="inputsLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="target">
         <property name="text">
          <string>&amp;Translation:</string>
         </property>
         <property name="buddy">
          <cstring>targetLed</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="targetLed">
         <property name="whatsThis">
          <string>This is the phrase in the target language corresponding to the source phrase.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="source">
         <property name="text">
          <string>S&amp;ource phrase:</string>
         </property>
         <property name="buddy">
          <cstring>sourceLed</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="definitionLed">
         <property name="whatsThis">
          <string>This is a definition for the source phrase.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="sourceLed">
         <property name="whatsThis">
          <string>This is the phrase in the source language.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="definition">
         <property name="text">
          <string>&amp;Definition:</string>
         </property>
         <property name="buddy">
          <cstring>definitionLed</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="phraseList">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="newBut">
       <property name="whatsThis">
        <string>Click here to add the phrase to the phrase book.</string>
       </property>
       <property name="text">
        <string>&amp;New Entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBut">
       <property name="whatsThis">
        <string>Click here to remove the entry from the phrase book.</string>
       </property>
       <property name="text">
        <string>&amp;Remove Entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingsBut">
       <property name="text">
        <string>Settin&amp;gs...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBut">
       <property name="whatsThis">
        <string>Click here to save the changes made.</string>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBut">
       <property name="whatsThis">
        <string>Click here to close this window.</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>51</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sourceLed</tabstop>
  <tabstop>targetLed</tabstop>
  <tabstop>definitionLed</tabstop>
  <tabstop>newBut</tabstop>
  <tabstop>removeBut</tabstop>
  <tabstop>saveBut</tabstop>
  <tabstop>closeBut</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeBut</sender>
   <signal>clicked()</signal>
   <receiver>PhraseBookBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
